index.html

```diff
@@ -492,7 +492,7 @@ <h3>
     </p>
     <ul>
       <li>Ensures that only one [=digital credential=] interaction is active at
-      any time. This includes both via the:
+      any time. This includes via both the:
```

Comment:
A colon can only be used in this fashion when it could sensibly be replaced by a period (a/k/a "full stop").
```suggestion
      any time, including by use of either or both of the following:
```